nextCacheHandler.mjs

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable no-console */
+import { CacheHandler } from "@neshca/cache-handler";
+import createLruHandler from "@neshca/cache-handler/local-lru";
+
+// This file is currently in js module format because it is imported into next.config.mjs
+// Once Next.js config supports TypeScript this file can be converted to `ts` module format
+
+CacheHandler.onCreation(async () => {
+  // Create an in-memory Least Recently Used cache for use in PR review environments
+  const localHandler = createLruHandler({
+    maxItemsNumber: 1000, // 1000 items
+    maxItemSizeBytes: 1024 * 1024 * 50, // 50 MB
```

Comment:
do I read this like a possible memory space of 50,000 MB (50GB)?

Reply:
Yes, you're right that is the worse case scenario.  We can shrink them if need be but those limits will not be attained by caching nextjs pages.  They were the defaults provided by the package author.